server/middleware/validation.js

```diff
@@ -289,6 +290,14 @@ const validators = {
             return message || validationErrors.isValidWithinGivenDays(label, props);
         }
         return null;
+    },
+    isValidWithinPastGivenDays({ label, value, message, props }) {
```

Comment:
I know this is the current use case but would having the props part be `{ days: 30 }` be more explicit. Refering as a prop doesn't give us the context.

This would then become the following:

```suggestion
    isValidWithinPastGivenDays({ label, value, message, { days }) {
```

Reply:
Makes sense, would `isValidWithinGivenDays` also need changing to `days` so that they both work in the same way?

Reply:
I'd say so, would need to make sure that the fields are updating that use the `isValidWithinGivenDays` as well.